01-CodeReviewEx/Shop-Ex/Program.cs

```diff
@@ -12,9 +12,33 @@ static void Main(string[] args)
         {
             
          
+         Account[] myAccounts = new Account[10];
+            Account a = new Account();
+            a.principal = 1.0;
+            a.rate = 0.5;
+            myAccounts[0] = a;
+            a = new Account() { principal = 1.5, rate = 0.75 };
+            
+            double fee = calculateFee(myAccounts);
+            Console.WriteLine(fee);
+         
+        }
+
+        public static double calculateFee(Account[] accounts)
+        {
+            double totalFee = 0.0;
+            Account account;
```

Comment:
If you're going to implement[this](https://github.com/skopczynska/spio-to/pull/15/commits/d0c3f59707a3be34b5bd63b7a335ff68c3b99947#r1405374650), then creation of this object is redundant